lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -20,6 +20,8 @@ $package = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $
 $force = Get-AnsibleParam -obj $params -name "force" -type "bool" -default $false
 $version = Get-AnsibleParam -obj $params -name "version" -type "str"
 $source = Get-AnsibleParam -obj $params -name "source" -type "str"
+$username = Get-AnsibleParam -obj $params -name "username" -type "str"
```

Comment:
I would suggest using `source_username` and `source_password` so it is more clear this relates to the `source` parameter specifically.